contrib/node/src/python/pants/contrib/node/tasks/javascript_style.py

```diff
@@ -89,18 +93,32 @@ def _install_javascriptstyle(self, javascriptstyle_dir):
 
   def _run_javascriptstyle(self, target, javascriptstyle_bin_path, files, fix=False):
     args = [javascriptstyle_bin_path]
-    if fix:
+    if fix is True:
       self.context.log.info('Autoformatting is enabled for javascriptstyle.')
       args.extend(['--fix'])
     args.extend(files)
-    result, node_run_command = self.execute_node(
-      args=args,
-      workunit_name=target.address.reference(),
-      workunit_labels=[WorkUnitLabel.PREP])
-    if result != 0 and not self.get_options().fail_slow:
-      raise TaskError('Javascript linting failed: \n'
-                      '{} failed with exit code {}'.format(node_run_command, result))
-    return result
+    
+    working_directory = target.target_base
+    ignore_path = self.get_ignore_path(target)
```

Comment:
are we only getting the first ignore_path here?  I am not sure I understand the logic.  This seems to be saying if there is any .gitignore anywhere under the target, only lint that directory?